Problem2.py

```diff
@@ -0,0 +1,44 @@
+class Solution:
+    def threeSum(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: List[List[int]]
+        """
+#       Starting with answer as set to store only unique triplet
+        res = set()
+        
+#       If length is less than 3 than we can't create triplet
+        if len(nums) < 3:
+            return res
+#       Sorting the number because, sorting time complexity: O(n) and algorithm takes more than that time complexity.
```

Comment:
sorting time complexity is o(nlogn) not o(n)

Reply:
Got it @kaurramanpreet . Wrote it by mistake. Can you also review Problem 3 that I added recently?